modules/social_features/social_event/social_event.module

```diff
@@ -60,6 +61,13 @@ function social_event_theme(): array {
   ];
 }
 
+/**
+ * Implements hook_entity_bundle_info_alter().
+ */
+function social_event_entity_bundle_info_alter(array &$bundles): void {
```

Comment:
@nechai Please add explanation comment why this is needed.

Reply:
Added